lms/djangoapps/verify_student/services.py

```diff
@@ -188,46 +168,51 @@ def user_status(cls, user):
         If the verification process is still ongoing, returns 'pending'
         If the verification has been denied and the user must resubmit photos, returns 'must_reverify'
 
-        This checks initial verifications
-        """
-        status = 'none'
-        error_msg = ''
+        This checks most recent verification
+        """
+        # should_display only refers to displaying the verification attempt status to a user
+        # once a verification attempt has been made, otherwise we will display a prompt to complete ID verification.
+        user_status = {
+            'status': 'none',
+            'error': '',
+            'should_display': True,
+        }
+
+        # We need to check the user's most recent attempt.
+        try:
+            attempts = IDVerificationAggregate.objects.filter(user=user).order_by('-updated_at')
+            attempt = attempts[0].content_object
+        except IndexError:
```

Comment:
I don't love using exceptions as flow control, but I see it is how the code was previously structured..